### PR TITLE
Fixed failing tests in test_patch.py

### PR DESCRIPTION
**Pull request by jesper-friis**
# Description
Due to resent updates in EMMO, several of the tests in test_patch.py were failing.

We should consider to test against our test ontology instead of EMMO to avoid such failures.

## Type of change

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.
- [x] Test update.

## Checklist

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: tests/ontopy_tests/test_patch.py [hunk 1]
```diff
@@ -44,37 +41,36 @@
 assert emmo.Atom["altLabel"] == ["ChemicalElement"]
 
 
+assert emmo.Atom.is_defined == False
```
Comment (francescalb) [inline tests/ontopy_tests/test_patch.py:44]: Is_defined should be tested as part of test_ontopy as it is defined there.

**Review by francescalb — CHANGES_REQUESTED** [review]
left one comment

File: tests/ontopy_tests/test_patch.py (hunk 1 above)
Comment (jesper-friis) [reply tests/ontopy_tests/test_patch.py:44]: We are testing the `is_defined` property of Atom, which is patched. The implementation in ontology is deprecated.

**Review by francescalb — COMMENTED** [review]
Very good, but please add an issue about the tests that are going to be added back.

**Pull request merged**